astra/src/test/java/com/slack/astra/testlib/SpanUtil.java

```diff
@@ -217,4 +217,18 @@ public static List<Trace.Span> makeSpansWithTimeDifference(
     }
     return result;
   }
+
+  public static Trace.Span makeSpansCustomKeywordTags(String key, String value, int spanNumber) {
+
+    List<Trace.KeyValue> tags = new ArrayList<>();
+    // Set alerts tag
```

Comment:
nit: the key here isn't just alerts now, so I think this comment should be rm'd or updated.